File: c/9.c
// imports
#include <stdio.h>
#include <stdlib.h>
int main(){
    // get len input
    int len;
    int width;
    scanf("%d", &len);
    scanf("%d", &width);
    // work out area
    int area = len * width;
    // log the result
    printf("The area is %d", area);
}

File: kotlin/9.kt
fun main() {
    // lets rectange the area
    val length: Float? = readln().toFloatOrNull();
    val width: Float? = readln().toFloatOrNull();
    if (length != null && width != null) {
        println("Area of rectangle is ${length * width}");
    } else {
        println("Invalid input"); // 10/10 error handling
    }
}